keras/src/layers/preprocessing/rescaling_test.py

```diff
@@ -84,3 +84,20 @@ def test_rescaling_with_channels_first_and_vector_scale(self):
         x = np.random.random((2, 3, 10, 10)) * 255
         layer(x)
         backend.set_image_data_format(config)
+
+    def test_numpy_args(self):
```

Comment:
Please exclude the NumPy backend from this test since it uses `fit()`.

Reply:
Fixed!